purview/api/v1.py

```diff
@@ -17,3 +18,13 @@ def about():
     :return: A JSON object with version info
     """
     return jsonify({'version': version})
+
+
+@api_v1.route('/<resource>/<id>')
+def get_resource(resource, id):
```

Comment:
I just realized `id` is overshadowing a built-in function. This should probably be renamed to `uid` or something similar.